go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/spf13/afero v1.1.2
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
```

Comment:
Is it expected?

Reply:
Unfortunately https://github.com/grafana/xk6-webcrypto/pull/64#discussion_r1519850102